webpack/assets/javascripts/react_app/components/CPUCoresInput/CPUCoresInput.stories.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import CPUCoresInput from './CPUCoresInput';
+
+export default {
+  title: 'Components|CPUCoresInput',
+  component: CPUCoresInput,
+  parameters: {
+    centered: { disable: true },
+  },
+};
+
+export const UseCPUorCoresInput = () => <CPUCoresInput label="CPUs" />;
```

Comment:
Looks like `UseCPUorCoresInput` doesn't work well with the storybook engine, see how it resolve in the left menu.

Reply:
What do you mean doesn't work well? where can I see it? where is this left menu?

Reply:
Ok I think I got it now :)